clients/client-shell/cmds/root/log.go

```diff
@@ -0,0 +1,28 @@
+package root
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// Logger is the default log.Logger of the commands
+	Logger = &logrus.Logger{
+		Out:       os.Stderr,
+		Formatter: new(logrus.TextFormatter),
+		Hooks:     make(logrus.LevelHooks),
+		Level:     logrus.ErrorLevel,
+	}
+)
+
+// setup log output based on --verbose flag
+func setUpLogs(enable bool) {
+	Logger.SetFormatter(&logrus.TextFormatter{
+		TimestampFormat: "02-01-2006 15:04:05",
+		FullTimestamp:   true,
```

Comment:
For a command-line tool, I think timestamps are not required.  Can this be configured to omit them?

Reply:
Yes, we can omit it, by default it will show the time(in seconds) since the start of the program along with each log